### PR TITLE
Add TypeScript declaration file

### PR DESCRIPTION
**Pull request by Joshua-Miles**
qa_req 0 (doesn't affect functionality at all)

### CODE REVIEW

File: localize.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function _js(origString: string, ...replacements: string[]): string;
+export function ___p(
+   number: number,
+   singularString: string,
+   pluralString: string,
+   ...replacements: string[]
+): string;
+export function setTranslations(translations: any): void;
```
Comment (danielbeardsley) [inline localize.d.ts]: I think this is just `Record<string, string>`
```suggestion
export function setTranslations(translations: Record<string, string>): void;
```

**Comment by danielbeardsley** [comment]
CR :+1: 

Nice!

**Comment by rjmccluskey** [comment]
CR 🍻

**Comment by danielbeardsley** [comment]
Published as 1.1.1: https://www.npmjs.com/package/@ifixit/localize/v/1.1.1

**Pull request merged**